cogs/track_react.py

```diff
@@ -88,6 +88,18 @@ def parse_codeblocks(message: str) -> str:
                 lines.append(line)
         return "\n".join(lines)
 
+    @staticmethod
+    def parse_url(message: str) -> str:
+        """Return a message with URLs removed"""
+        lines = []
+        pattern = r"(https?://\S+)"
```

Comment:
This can be a module constant and compiled once at module import vs having to compile it for every line of every message.